os_update-formula/os-update/init.sls

```diff
@@ -34,8 +40,19 @@ os-update_config:
         {%- endif %}
         {{ option | upper }}: '"{{ value }}"'
         {%- endfor %}
+    {%- if opts['test'] %}
+    - ignore_if_missing: true
+    {%- endif %}
```

Comment:
Nitpicking: I'd expect that `file.keyvalue` knows how to behave when running in test mode, therefore I'm surprised to see this condition.

Since that's only nitpicking, feel free to merge with or without dropping these 3 lines.

Reply:
Unfortunately it doesn't, it will result in a failed test state run if the file does not exist.

Reply:
For reference, I explained it in the commit message here: https://github.com/openSUSE/salt-formulas/pull/20/commits/352936dd40da10a5308b4ea92355ab86d7b95855